app/test_main.py

```diff
@@ -1,3 +1,66 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "input_value_cat,input_value_dog,expected_value",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="does not work correctly with the value zero"
+            ),
+            pytest.param(
+                100,
+                100,
+                [21, 17],
+                id="does not work correctly with a large value"
```

Comment:
it's a valid case too.

Creme Puff is the oldest cat ever at 38 years and 3 days old. She was born on August 3, 1967, and died on August 6, 2005.
Creme Puff was owned by Granpa Rex Allen's owner and started her day with bacon and eggs, asparagus, broccoli, and coffee with heavy cream each morning.